src/nih_wayfinding/app/scripts/routing/directions-service.js

```diff
@@ -33,11 +33,38 @@
             params.destination = destination;
             return $http.post(directionsUrl, params, {});
 */
-            return $timeout(function () {
-                var res = RoutingResponseStub;
-                TurnAmenities.attach(res);
-                return res;
-            }, 100, false);
+            if (!(_.isArray(origin) && origin.length === 2)) {
+                return;
+            }
+            if (!(_.isArray(destination) && destination.length === 2)) {
+                return;
+            }
+            var now = new Date();
+            // This is the minimum param list (+ toPlace,fromPlace) to make a OTP plan API call
+            var defaults = {
+                mode: 'WALK',
+                time: now.getHours() + ':' + now.getMinutes(),
+                date: defaultDate(),
+                arriveBy: false,
+                wheelchair: false,
+                showIntermediateStops: false
+            };
+            var params = angular.extend({}, defaults, options);
+            // Swap, OTP uses [lon,lat]
```

Comment:
Maybe it would make more sense to store as [lon, lat], since both OTP and GeoJSON use that format.